h/static/scripts/group-forms/components/CreateGroupForm.tsx

```diff
@@ -6,11 +6,21 @@ function Star() {
   return <span className="text-brand">*</span>;
 }
 
-function CharacterCounter({ value, limit }: { value: number; limit: number }) {
+function CharacterCounter({
```

Comment:
I expect you'll want to extract this into its own module in future for reuse elsewhere. That should be easy to do when needed.